application/database/migrations/20210125173740_add_aprovado_to_status_list.php

```diff
@@ -7,7 +7,7 @@ public function up()
         $configurationSql = "
             SELECT valor
             FROM configuracoes
-            WHERE idConfig = 12
+            WHERE idConfig = 13
```

Comment:
Qual o motivo de ficar 13?

Reply:
Porque todas são numeradas, se não numera da problema. 

Fiz um teste alterando as remunerações. 

Como já vem assim de outras datas se for remover de todos iria da problema.

E fazer uma migration pra corrigir num sei se é certo